src/app/item-popup/WeaponDeepsightInfo.tsx

```diff
@@ -9,18 +14,39 @@ export function WeaponDeepsightInfo({ item }: { item: DimItem }) {
   const record = item.patternUnlockRecord;
   const relevantObjectives = record?.objectives.filter((o) => !o.complete);
 
+  const defs = useD2Definitions()!;
+  const harmonizerIcon = defs.InventoryItem.get(DEEPSIGHT_HARMONIZER).displayProperties.icon;
+
   if (!relevantObjectives?.length) {
     return null;
   }
 
+  const harmonizable = isHarmonizable(item);
+  const harmonizerTooltipText = item.tooltipNotifications?.map((t) => t.displayString);
+  const harmonizableTooltip = (
+    <>
+      <p>{harmonizerTooltipText}</p>
+      <p>
+        Filter with <code>deepsight:harmonizable</code>
```

Comment:
this would need i18n if we're committed to it

Reply:
+1 on translating this